functional/OpenJcePlusTests/test.xml

```diff
@@ -16,13 +16,16 @@
 
 <project name="Launcher script for OpenJcePlusTests" default="help" basedir=".">
 	<target name="help">
-		<echo>Ant script to run the tests for ibm.jceplus.junit.TestAll.
+		<echo>Ant script to run the tests for ibm.jceplus.junit.TestMemStressAll,
+		ibm.jceplus.junit.TestMultiThread,
+		ibm.jceplus.junit.TestMultithreadFIPS,
+		ibm.jceplus.junit.TestAll.
 		</echo>
 	</target>
 
 	<target name="launch_test">
-		<echo message="Running ibm.jceplus.junit.TestAll in ${user.dir}" />
-		<echo message="Command: ${TEST_JAVA} -cp junit4.jar:hamcrest-core.jar:bcprov-jdk18on.jar:openjceplus-tests.jar --add-exports=java.base/sun.security.util=ALL-UNNAMED --add-exports=openjceplus/com.ibm.crypto.plus.provider.ock=ALL-UNNAMED ibm.jceplus.junit.TestAll" />
+		<echo message="Running ibm.jceplus.junit.TestMemStressAll, ibm.jceplus.junit.TestMultiThread, ibm.jceplus.junit.TestMultithreadFIP and ibm.jceplus.junit.TestAll in ${user.dir}" />
+		<echo message="Command: ${TEST_JAVA} -cp junit4.jar:hamcrest-core.jar:bcprov-jdk18on.jar:openjceplus-tests.jar --add-exports=java.base/sun.security.util=ALL-UNNAMED --add-exports=openjceplus/com.ibm.crypto.plus.provider.ock=ALL-UNNAMED ibm.jceplus.junit.TestMemStressAll/TestMultiThread/TestMultiThreadFIPS/TestAll" />
```

Comment:
The purpose of this echo is for users to run the command directly if needed. Please put a valid command for each run. And they should be moved into `test` target.

Reply:
updated